src/components/SummaryTable.test.js

```diff
@@ -91,7 +81,7 @@ const testMandatoryAnswers = {
   operator_first_name: "John",
   operator_last_name: "Appleseed",
   operator_company_name: "Company name",
-  operator_companies_house_number: "AA123456",
```

Comment:
This was due to me pulling develop before 2 changes went out. I have remerged these in manually so hopefully its better. Note that the prettier defaults have changed with the newer version (v1>v2). I will put a PR in for develop with just prettier upgraded. This will make it easier to see the changes in the new version.